src/Number.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Number.h"
+
+namespace faker
+{
+
+/**
+ * @brief A random device used to seed the Number module pseudo-random generator.
```

Comment:
I dont think we need those docs about randomDevice and pseudoRandomGenerator, we need to be careful to write only essential comments/docs because it would be hard to modify and read later if we make many of them

Reply:
You are right, it is best to document only the essential things, otherwise the important stuff gets lost in the noise, you can go ahead and clean this